libs/utils/trace_analysis.py

```diff
@@ -340,10 +342,10 @@ def _plotTaskSignals(self, df, axes, task_name, tid, signals, is_last=False):
         if 'sched_overutilized' in signals:
             self.plotOverutilized(axes)
 
-    def _plotTaskResidencies(self, df, axes, task_name, signals, is_last=False):
-        axes.set_title('CPUs residency (green: LITTLE, red: big)');
+    def _plotTaskResidencies(self, axes, tid, signals, is_last=False):
+        util_df = self.trace.df('sched_load_avg_task')
         axes.set_xlim(0, self.trace.time_range);
```

Comment:
This was already present before this patch, but why are there two calls to `set_xlim()`?